src/scripts/annotate.pl

```diff
@@ -423,22 +438,36 @@ sub is_bad_fusion{
 	$second_bp->{tname} = normalizeChromosomeName($seq_ids[0], $second_bp->{tname});
 
 	# Ensure that the breakpoint chromosome names match
-	next if(!$internal && is_bad_fusion($first_bp->{tname}, $first_bp->{tpos}, $second_bp->{tname}, $second_bp->{tpos}));
+	if(!$internal){
+		#if(is_knownfusiongenepair(\@genes1, \@genes2)){
+ 		#	next if(is_bad_fusion($first_bp->{tname}, $first_bp->{tpos}, $second_bp->{tname}, $second_bp->{tpos}));
+		#}
+		#else{
+			#if(is_knownbadfusionpair(\@genes1, \@genes2)) next;#if genename NA in unfiltered.fusion.txt, it does not match bad_fusions; because NA -> chr:pos in bad_fusions; e.g. ATAD3A-ATAD3B in black fusion list; any ATAD3A-ATAD3B in unfiltered.fusion.txt will be filtered
+			#else{#GeneA-GeneB or GeneA-NA or NA-NA in unfiltered.fusion.txt is not in black fusion list
```

Comment:
```suggestion
```

Reply:
Since there is no change, consider just restoring the original line: <https://github.com/stjude/CICERO/pull/66/files#diff-94f5ff5d051cda255c8ba3357549025ef09976f5dc6886fb64bbc93f97ccbae2L426>.